go.mod

```diff
@@ -1,107 +1,103 @@
 module github.com/grafana/xk6-client-tracing
 
-go 1.21
+go 1.23.0
```

Comment:
Is there a specific reason for setting the Go version to 1.23 while the Go `toolchain` is set to 1.24? What do you think about setting the Go version to 1.24 as well?

Reply:
I had this as `go 1.24` at some point but that was causing issues compiling into k6 _I think_ because k6 or some other library required go 1.23 https://github.com/grafana/k6/blob/master/go.mod#L3 

I removed this line and let `go mod tidy` figure out the minimum go version which is where this `go 1.23` came from. The toolchain I also tried to remove and have auto-add and it kept the go 1.24. Admittedly I'm not sure if that's because that is the minimum or just because I have go 1.24 though